cluster/manifests/wiz/connector-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       volumes:
         - name: connector-data
           secret:
-            secretName: wiz-connector-connector
```

Comment:
This change doesn't look related to the title/description. We would also need to update the name of  the `Secret` resource.